site/content/how-to/monitoring/troubleshooting.md

```diff
@@ -9,7 +9,73 @@ docs: "DOCS-1419"
 
 This topic describes possible issues users might encounter when using NGINX Gateway Fabric. When possible, suggested workarounds are provided.
 
-### NGINX fails to reload
+### **General Troubleshooting**
+
+When attempting to diagnose a problem or get support, there are a few important data points that can be collected to help with understanding what issues may exist.
+
+#### Resource Status
+
+To get the status of a resource, use `kubectl describe`. For example, to check the status of the `coffee` HTTPRoute:
+
+```shell
+kubectl describe httproutes.gateway.networking.k8s.io coffee [-n namespace]
+```
+
+```text
+...
+Status:
+  Parents:
+    Conditions:
+      Last Transition Time:  2024-05-31T16:22:26Z
+      Message:               The route is accepted
+      Observed Generation:   1
+      Reason:                Accepted
+      Status:                True
+      Type:                  Accepted
+      Last Transition Time:  2024-05-31T16:22:26Z
+      Message:               All references are resolved
+      Observed Generation:   1
+      Reason:                ResolvedRefs
+      Status:                True
+      Type:                  ResolvedRefs
+    Controller Name:         gateway.nginx.org/nginx-gateway-controller
+    Parent Ref:
+      Group:         gateway.networking.k8s.io
+      Kind:          Gateway
+      Name:          gateway
+      Namespace:     default
+      Section Name:  http
+```
+
+If a resource has any errors relating to its configuration or relation to other resources, it is likely that those errors will be contained within the status.
+
+#### Events
+
+Events may be created by NGINX Gateway Fabric or other Kubernetes components that could indicate system or configuration issues. To see events:
+
+```shell
+kubectl get events [-n namespace]
+```
+
+#### Logs
+
+Logs of the NGINX Gateway Fabric control plane and data plane can contain information that isn't otherwise reported in status or events. These could include errors in processing or passing traffic.
+
+To see logs for the control plane container (replacing the name of the deployment if necessary):
```

Comment:
since users can scale pods, how would it work when there > 1 pod?

Reply:
Good point, I'll use pod instead of deployment